src/relayserver/runServer.ts

```diff
@@ -68,6 +69,8 @@ const ethereumNodeUrl: string = argv.ethereumNodeUrl ?? config.ethereumNodeUrl ?
 const workdir: string = argv.workdir ?? config.workdir ?? error('missing --workdir')
 const devMode: boolean = argv.devMode ?? config.devMode ?? error('missing --devMode')
 const debug: boolean = argv.debug ?? config.debug ?? error('missing --debug')
+const registrationBlockRate: string = argv.registrationBlockRate ?? config.registrationBlockRate?.toString()
+console.log('wtf is registrationBlockRate?', registrationBlockRate)
```

Comment:
please be respectful with "wtf". if you do need it, put it in "debug"

Reply:
or better: when the server starts, dump its complete active configuration.

Reply:
Or better, have a git hook not to commit it